bin/cli.ts

```diff
@@ -1,16 +1,16 @@
-#!/usr/bin/env node
-const fs = require('fs');
+#! node
```

Comment:
Could you revert this change? `/usr/bin/env node` is much safer than calling `node` directly